### PR TITLE
Hello Rails - Init

### PR DESCRIPTION
**Pull request by lu-jim**
## Purpose:
- Implement a connection between a Ruby on Rails back-end and React front-end.
- Understand the pros and cons of different approaches to connecting Ruby on Rails back-end with React front-end.

## Features:
- Created base project with Rails
- Installed Webpacker and rails-react
- Configured rails API for the random greeting
- Built the Greeting and App components to connect to the rails API

### CODE REVIEW

File: app/views/static/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>Hello Rails React</h1>
+<%= react_component('App')%>
```
Comment (andersonlebon) [inline app/views/static/index.html.erb]: - [x] Kindly note the  `<h1>Hello Rails React</h1>` is what is being rendered but the requirement says that is the random greeting that should be rendered. Kindly fix it. 

image

Comment (andersonlebon) [inline app/views/static/index.html.erb]: - [x] Kindly note the  `<h1>Hello Rails React</h1>` is what is being rendered but the requirement says that is the random greeting that should be rendered. Kindly fix it.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @lu-jim 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly provide a professional`README` by following the microverse `readme` template.

image


- [ ] Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mmsesay — APPROVED** [review]
Hi @lu-jim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**